pdns/dnsdist-lua-rules.cc

```diff
@@ -0,0 +1,1206 @@
+/*
```

Comment:
Although we call this rules, it are selectors... maybe we should rename this in the future :)